addons/skin.confluence/720p/includes.xml

```diff
@@ -14,8 +14,29 @@
 	<include file="IncludesHomeMenuItems.xml" />
 	<include file="IncludesPVR.xml" />
 	<include file="IncludesBackgroundBuilding.xml" />
+	<!-- CONSTANTS -->
 	<constant name="FanartCrossfadeTime">500</constant>
 	<constant name="IconCrossfadeTime">400</constant>
+	<constant name="DepthDialogue">0.55</constant>
+	<constant name="DepthDialogue+">0.60</constant>
+	<constant name="DepthDialogue-">0.50</constant>
+	<constant name="DepthMax">0.65</constant>
+	<constant name="DepthOSD">0.4</constant>
+	<constant name="DepthOSD+">0.45</constant>
+	<constant name="DepthOSD-">0.35</constant>
+	<constant name="DepthOSDPopout">0.5</constant>
+	<constant name="DepthContent">0</constant>
```

Comment:
0 doesn't mean no depth but screen level. If you have a control group that's f.e. using a different level but one single control within that group should be pushed back to level 0, this constant may be used